### PR TITLE
Check if saved window size is at least the minimum allowed

### PR DESCRIPTION
**Pull request by ImUrX**

### CODE REVIEW

**Review by Louka3000 — APPROVED** [review]
lgtm

File: gui/src-tauri/src/state.rs [hunk 1]
```diff
@@ -73,7 +75,11 @@ impl WindowState {
 			window.unmaximize()?;
 		}
 
-		window.set_size(LogicalSize::new(self.width, self.height))?;
+		if self.width > util::MIN_WINDOW_SIZE_WIDTH
+			&& self.height > util::MIN_WINDOW_SIZE_HEIGHT
+		{
+			window.set_size(LogicalSize::new(self.width, self.height))?;
+		}
```
Comment (TheDevMinerTV) [inline gui/src-tauri/src/state.rs:82]: Why not just use `std::cmp::max()`?

Comment (ImUrX) [reply gui/src-tauri/src/state.rs:82]: the idea is to not set size if somehow it saved an incorrect one

Comment (TheDevMinerTV) [reply gui/src-tauri/src/state.rs:82]: nvm, you're right

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**